Sources/Orbit/Foundation/Elevations/Elevation.swift

```diff
@@ -0,0 +1,165 @@
+import SwiftUI
+
+/// Use elevation to bring content closer to users.
+///
+/// Elevation levels with higher numbers are usually visually closer to the user.
+public enum Elevation {
+    case level1
+    case level2
+    case level3
+    case level4
+}
+
+public extension View {
+
+    /// Elevates the view to make it more prominent.
```

Comment:
Suggestion: Adding a discussion paragraph about the performence and possibilities to adjust:

Multiple views can be elevated at once by using this modifier on a container view. Either for performance reasons, but also to be able to use `compositingGroup()` modifier to decide whether the elevation will be applied on each view separately or on their merged composition.

An elevation in child views can be disabled by using ``IsElevationEnabledKey`` environment key.